src/templates/ios.handlebars

```diff
@@ -6,8 +6,7 @@
         {{description}}
     {{/if}}
     public enum
-    {{removeSpaces this.name}}
-    {
+    {{removeSpaces this.name}}: Sendable {
```

Comment:
This could run into problems if one of the `let` constants is not a Sendable type. i think so far, none are, but not sure that's guaranteed with the current set.